OJP/OJP_Trips.xsd

```diff
@@ -753,14 +800,9 @@
 					<xs:documentation>Number how often the access feature occurs in this PathLink</xs:documentation>
 				</xs:annotation>
 			</xs:element>
-			<xs:element name="From" type="PathLinkEndStructure" minOccurs="0">
-				<xs:annotation>
-					<xs:documentation>Designations of level and place where this PathLink starts.</xs:documentation>
-				</xs:annotation>
-			</xs:element>
-			<xs:element name="To" type="PathLinkEndStructure" minOccurs="0">
```

Comment:
Why does this fragment from the code indroduced by PR #239 shine up here (*From* and first line of *To*)? 
(having selected *Changes from all commits*)

PR #239 should remain intact when doing the merge.

Reply:
@trurlurl : merge problem. #239 must be in it 100% as it is merged. I check.

Reply:
resolved